rainbow-me/rainbow
# Update OP Round 2 Rewards Sheet

Fixes RNBW-4698

## What changed (plus any additional context for devs)
Adjusted OP Rewards sheet to match Round 2 designs/spec.

## Screen recordings / screenshots

<img width="487" alt="Screenshot 2023-10-10 at 4 27 17 PM" src="https://github.com/rainbow-me/rainbow/assets/6284525/5434a70a-2bb3-4785-90a8-6aeaad3f72f3">
<img width="487" alt="Screenshot 2023-10-10 at 4 26 55 PM" src="https://github.com/rainbow-me/rainbow/assets/6284525/0916244c-7a61-4d77-859c-15c593ec8b60">

## What to test
1. Enable $OP Rewards feature flag in settings
2. Verify that the card is featured underneath NFTs/Mints in the discover page
3. Click on the card and verify the Rewards sheet shows
4. Report any css/alignment issues if found
